recipes/server.rb

```diff
@@ -17,15 +17,24 @@
 # limitations under the License.
 #
 
+Chef::Recipe.send(:include, MariaDB::Helper)
 case node['mariadb']['install']['type']
 when 'package'
-  include_recipe "#{cookbook_name}::repository"
+  if use_os_native_package?(
```

Comment:
Please check the above use_os_native_package? method in helper. In there, it has logic to handle this situation.